.github/workflows/python-app.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
+
+jobs:
+  lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Install ldap dependencies
+      run: sudo apt-get install libldap2-dev libsasl2-dev
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
```

Comment:
Python 3.6 is in it's last year of security fixes.

Reply:
I'm aware. I'm moving apps up to 3.8, but wanted to test the CI on the version running on okd, but I suppose it's safe to just jump to 3.8 with us having a dev instance

Reply:
YOLO ops eh?